Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserTableModel.cpp

```diff
@@ -24,13 +24,23 @@ namespace AzToolsFramework
             AZ_Assert(
                 m_filterModel,
                 "Error in AssetBrowserTableModel initialization, class expects source model to be an AssetBrowserFilterModel.");
+
             QSortFilterProxyModel::setSourceModel(sourceModel);
 
-            connect(m_filterModel, &QAbstractItemModel::rowsInserted, this, &AssetBrowserTableModel::UpdateTableModelMaps);
-            connect(m_filterModel, &QAbstractItemModel::rowsRemoved, this, &AssetBrowserTableModel::UpdateTableModelMaps);
-            connect(m_filterModel, &QAbstractItemModel::layoutChanged, this, &AssetBrowserTableModel::UpdateTableModelMaps);
+            auto checkUpdateModelTimer = [&]()
```

Comment:
nit: does this need to be a lambda? Would it be more readable if it was just another member like UpdateTableModelMaps?

Reply:
I want to say a lambda is okay because this shouldn't (not just won't) be called anywhere else. Probably wouldn't hurt to have a comment here though.

Those are just my thoughts. If it's confusing or bad style to have it here then I'm fine with creating a function for it.

Reply:
I'm fine with either a private function or a lambda with a comment :)

Reply:
Done! Added a `const int` local to the lambda so it doesn't need to be captured or copied, and for tidiness.